ironfish-rust-nodejs/src/structs/note_encrypted.rs

```diff
@@ -9,20 +9,31 @@ use napi::bindgen_prelude::*;
 use napi::JsBuffer;
 use napi_derive::napi;
 
+use ironfish_rust::merkle_note::NOTE_ENCRYPTION_KEY_SIZE;
+use ironfish_rust::note::ENCRYPTED_NOTE_SIZE;
 use ironfish_rust::MerkleNote;
 
 use crate::to_napi_err;
 
 #[napi]
-pub const ENCRYPTED_NOTE_LENGTH: u32 = 296;
+pub const NOTE_ENCRYPTION_KEY_LENGTH: u32 = NOTE_ENCRYPTION_KEY_SIZE as u32;
+
+#[napi]
+pub const ENCRYPTED_NOTE_PLAINTEXT_LENGTH: u32 = ENCRYPTED_NOTE_SIZE as u32;
+
+#[napi]
+pub const MAC_LENGTH: u32 = 16;
+
+#[napi]
+pub const ENCRYPTED_NOTE_LENGTH: u32 =
+    NOTE_ENCRYPTION_KEY_LENGTH + ENCRYPTED_NOTE_PLAINTEXT_LENGTH + MAC_LENGTH + 96;
 //  32 value commitment
 //+ 32 note commitment
 //+ 32 ephemeral public key
-//+ 100 encrypted note
-//+ 16 aead MAC
-//+ 64 note encryption keys
+//+ 104 encrypted note
 //+ 16 aead MAC
-//= 264 bytes
+//+ 80 note encryption keys
```

Comment:
`NOTE_ENCRYPTION_KEY_SIZE` is `ENCRYPTED_SHARED_KEY_SIZE` (64) + `aead::MAC_SIZE` (16). these should stay split out if we're going to split out the MAC size from the encrypted note as well

Reply:
Decided to merge both together